web/scripts/app.js

```diff
@@ -201,6 +202,23 @@ function redraw() {
 }
 
 
+function _search_from(query, rowid) {
+    var data = grid.getData();
+    var numrows = data.getLength();
+    var next = false;
+
+    for (var i = rowid + 1; i < numrows; ++i) {
+        var row = data.getItem(i);
+        if (row.searchname == query || row.searchname.indexOf(query) >= 0) {
```

Comment:
These are sort of the same thing. With this new behavior, we probably just want `if (row.searchname.indexOf(query) >= 0)`.